roles/redis-collection/tasks/main.yml

```diff
@@ -0,0 +1,63 @@
+---
+
+- name: Get status code of the test
+  command: redis-cli hget uperf-test status
+  register: status_code
+
+- debug:
+    msg: "The status code for the test with uuid {{uuid}} and parameters as {{uperf.test_types}}-{{uperf.protos}}-{{uperf.sizes}}-{{uperf.nthrs}} is {{status_code.stdout}}"
+
+
+- block:
+
+  - block:
+    - name: Get Server Pods
+      k8s_facts:
+        kind: Pod
+        api_version: v1
+        namespace: '{{ operator_namespace }}'
+        label_selectors:
+          - type = uperf-bench-server-{{ trunc_uuid }}
+      register: server_pods
+
+    - name: Pod names - to clean
+      set_fact:
+        clean_pods: |
+            [
+            {% for item in server_pods.resources %}
+              "{{ item['metadata']['name'] }}",
+            {% endfor %}
+            ]
+
+    - name: Cleanup run
+      k8s:
+        kind: pod
+        api_version: v1
+        namespace: '{{ operator_namespace }}'
+        state: absent
+        name: "{{ item }}"
+      with_items: "{{ clean_pods }}"
+
+  - k8s_status:
+      api_version: ripsaw.cloudbulldozer.io/v1alpha1
+      kind: Benchmark
+      name: "{{ meta.name }}"
+      namespace: "{{ operator_namespace }}"
+      status:
+        state: Failed
+        complete: true
+
+  - name: Delete benchmark
+    k8s:
+      kind: benchmark
+      api_version: ripsaw.cloudbulldozer.io/v1alpha1
+      namespace: '{{ operator_namespace }}'
+      state: absent
+      name: "uperf-benchmark"
```

Comment:
This is explicitly calling out uperf-benchmark, however it could be any benchmark, right?